docs/isolated-cpu-affinity-transition.md

```diff
@@ -0,0 +1,148 @@
+## Isolated CPU affinity transition
+
+The introduction of the kernel commit 46a87b3851f0d6eb05e6d83d5c5a30df0eca8f76
+in 5.7 has affected a deterministic scheduling behavior by distributing tasks
+across CPU cores within a cgroups cpuset. It means that some runc operations
+like `runc exec` might be impacted under some circumstances, by example when
+a container has been created within a cgroup cpuset entirely composed of
+isolated CPU cores usually sets either with `nohz_full` and/or `isolcpus`
+kernel boot parameters.
+
+Some containerized real-time applications are relying on this deterministic
+behavior and uses the first CPU core to run a slow thread while other CPU
+cores are fully used by the real-time threads with SCHED_FIFO policy.
+Such applications can prevent runc process from joining a container when the
+runc process is randomly scheduled on a CPU core owned by a real-time thread.
+
+Runc introduces a way to restore this behavior by adding the following
+annotation to the container runtime spec (`config.json`):
+
+`org.opencontainers.runc.exec.isolated-cpu-affinity-transition`
```

Comment:
This list should be updated too
https://github.com/opencontainers/runc/blob/8e69225e2e5f25c46e79cf49935db362ead8f496/features.go#L67-L71

Reply:
Ok, do you know if CRI implementations like containerd are using this list actually to automatically exclude annotations ?

Reply:
Not yet